docs/notebooks/dwi_simulated_gp.ipynb

```diff
@@ -0,0 +1,685 @@
+   "cell_type": "markdown",
+   "source": "Define a method to get the appropriate signal representation",
+   "id": "25fc548550feb308"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-07-12T01:48:41.594953Z",
+     "start_time": "2024-07-12T01:48:41.589980Z"
+    }
+   },
+   "cell_type": "code",
+   "source": [
+    "def compute_raw_signal_representation(_dwi_data, _bvecs, _b0_mask, _voxel_idx):\n",
+    "\n",
+    "    # There is only one b0 volume in the simulated signal\n",
+    "    s0_voxel = _dwi_data[_voxel_idx[0], _voxel_idx[1], _voxel_idx[2], _b0_mask]\n",
+    "    s_voxel = _dwi_data[_voxel_idx[0], _voxel_idx[1], _voxel_idx[2], ~_b0_mask]\n",
+    "\n",
+    "    # Scale gradient vector values with DWI signal\n",
+    "    s_normal = s_voxel/s0_voxel\n",
+    "    s_dir_values = _bvecs[~_b0_mask, :] / s_normal[:, np.newaxis]\n",
+    "\n",
+    "    return s_dir_values"
+   ],
+   "id": "3abdf5122238f9c1",
+   "outputs": [],
+   "execution_count": 6
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "Obtain the appropriate representation for the DWI data",
+   "id": "9edff59aa8dadbcb"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-07-12T01:48:45.862760Z",
+     "start_time": "2024-07-12T01:48:45.860203Z"
+    }
+   },
+   "cell_type": "code",
+   "source": "signal_repr = compute_raw_signal_representation(dwi_data, gtab.bvecs, gtab.b0s_mask, voxel_idx)",
+   "id": "70d1630d58576a72",
+   "outputs": [],
+   "execution_count": 7
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "Define a method to plot the DWI data",
+   "id": "cf8e5fb998123a47"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-07-12T01:48:48.512196Z",
+     "start_time": "2024-07-12T01:48:48.509081Z"
+    }
+   },
+   "cell_type": "code",
+   "source": [
+    "def plot_dwi_scatter(_dwi_data):\n",
+    "\n",
+    "    # Plot the DWI signal as a 3D point cloud\n",
+    "    fig = plt.figure()\n",
+    "    ax = fig.add_subplot(111, projection=\"3d\")\n",
+    "\n",
+    "    _ = ax.scatter(_dwi_data[:, 0], _dwi_data[:, 1], _dwi_data[:, 2], c=\"blue\", marker=\"*\")\n",
+    "\n",
+    "    # Set labels\n",
+    "    ax.set_xlabel(\"X\")\n",
+    "    ax.set_ylabel(\"Y\")\n",
+    "    ax.set_zlabel(\"Z\")\n",
+    "    ax.set_title(\"Normalized dMRI signal\")\n",
+    "\n",
+    "    return fig"
+   ],
+   "id": "764dff2e2c5a9b8d",
+   "outputs": [],
+   "execution_count": 8
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "Plot the DWI data as a scatter plot",
+   "id": "95cfad7bd0b98fd6"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-07-12T01:48:51.227899Z",
+     "start_time": "2024-07-12T01:48:51.205608Z"
+    }
+   },
+   "cell_type": "code",
+   "source": [
+    "# Create an interactive plot\n",
+    "%matplotlib notebook\n",
+    "\n",
+    "fig = plot_dwi_scatter(signal_repr)\n",
```

Comment:
Shows two disks, so not quite right yet.

![signal_scatter_2](https://github.com/user-attachments/assets/7fe00852-2132-4f63-965d-a8df84b1002f)

![signal_scatter_1](https://github.com/user-attachments/assets/737e7e8e-7f67-4fcf-94dc-f03bd949e11a)